src/app/common/services/auth-service.service.ts

```diff
@@ -30,6 +37,18 @@ export class AuthService {
     this.router.navigate(['/login']);
```

Comment:
it will go back to `/clourses` page because `isLoggedSubject`  will have `true` value. At logout, you should clear user data, emit false on `isLoggedSubject` and navigate to `/` page. On '/' should be canActivate guard that will redirect to the login page if the user isn't logged in.